docs/en/observability/monitor-nginx-ml.asciidoc

```diff
@@ -0,0 +1,70 @@
+[discrete]
+[[monitor-nginx-ml]]
+== Part 5: Find anomalies in your nginx access logs
+
+The {integrations-docs}/nginx[Nginx Elastic integration] to help find unusual activity in your nginx access logs.
```

Comment:
Missing a word here. Maybe "use"? Also you lowercased nginx Elastic integration above.

```suggestion
Use the {integrations-docs}/nginx[nginx Elastic integration] to help find unusual activity in your nginx access logs.
```